roles/confluent.ksql/defaults/main.yml

```diff
@@ -1,13 +1,12 @@
 ---
 # More ksql variables are defined in the confluent.variables role
 
-### Boolean to enable cp-ansible's Custom Log4j Configuration
+### Boolean to reconfigure ksqlDB's logging with RollingFileAppender and log cleanup
```

Comment:
Boolean to reconfigure ksqlDB's logging with the RollingFileAppender and log cleanup functionality.